packages/@contentful--app-scripts/lib/upload/organization-api.js

```diff
@@ -0,0 +1,72 @@
+const inquirer = require('inquirer');
+const chalk = require('chalk');
+const ora = require('ora');
+
+async function fetchOrganizations(client) {
```

Comment:
Also using the error wrapper I was suggesting below and the plain client in CMA, this can become a couple of lines of code. You might not need this utility in the first place

Reply:
I actually already did that in the second PR about activation. But I moved it now here too